automation/taskcluster/schedule_snapshot_release.py

```diff
@@ -77,9 +77,9 @@ def as_slugid(name):
 def schedule():
     queue = taskcluster.Queue({'baseUrl': 'http://taskcluster/queue/v1'})
 
-    repository_github_http_url, branch, head_rev = calculate_git_references(ROOT)
+    repository_github_html_url, branch, head_rev = calculate_git_references(ROOT)
     params = {
-        'repository_github_http_url': repository_github_http_url,
+        'repository_github_html_url': repository_github_html_url,
```

Comment:
Nit: how come we're not renaming this consistently `html_url` like in https://github.com/mozilla-mobile/reference-browser/pull/410/ ?

Reply:
Nice catch, thanks!